clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -75,6 +59,28 @@ export const ConsentButtons = ({
         </Fragment>
       )}
     </div>
+    <div
+      className={`${
```

Comment:
Have you ever worked with the classNames npm package? (https://www.npmjs.com/package/classnames). I don't think we're using it on this project, but I bet it would make all of these conditional classes much easier. Give it a try if you can.

Reply:
Unfortunately we have to keep this codebase as tiny as possible since it ends up on the client's site and affects load time, so I hesitate adding a new convenience package like this.